user/xfsdump/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "xfsdump"
+pkgver = "3.2.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--enable-lib64=no"]
+# autoheader breaks definitions in include/config.h.in
+configure_gen = []
+# build system expects to be in the root of the source directory
+make_dir = "."
+make_install_args = ["PKG_ROOT_SBIN_DIR=/usr/bin"]
+hostmakedepends = ["gettext", "libuuid-devel"]
```

Comment:
why is libuuid-devel a host dependency?

Reply:
It's used by autoconf, but we aren't doing that so it can be removed.